plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java

```diff
@@ -400,6 +400,24 @@ public void doExport(StaplerRequest req, StaplerResponse res) throws Exception {
         export(res.getOutputStream());
     }
 
+    /**
+     * Export JSONSchema to URL
+     * @throws Exception
+     */
+    @RequirePOST
+    public void doNewSchema(StaplerRequest req, StaplerResponse res) throws Exception {
+
+        if (!Jenkins.getInstance().hasPermission(Jenkins.ADMINISTER)) {
+            res.sendError(HttpServletResponse.SC_FORBIDDEN);
+            return;
+        }
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+        export(out);
+
+        req.setAttribute("export", out.toString(StandardCharsets.UTF_8.name()));
+        req.getView(this, "viewExport.jelly").forward(req, res);
```

Comment:
This won't work for an IDE. This needs to be turned as application/json and cannot contain HTML.

If you want to you need two new URLs than one for `viewSchema` and one for `schema`

Also the currently `viewExport` only set to YAML Tokenization.